docs/SCD2-sparkcompute.md

```diff
@@ -56,6 +58,26 @@ After deduplicating, the resulting records will be:
 
 **Blacklist:** Blacklist for fields to ignore to compare when deduplicating the record.
 
+**Preserve Target** Preserve all the records from the scd2 target table when deduplicating.
+
+**Target Field** The name of the field that tells whether the record is coming from the scd2 target table. This field must be boolean.
+For example, suppose the following records are processed with Deduplicate and Preserve Target set to true, with Blacklist set to start_date, is_target:
```

Comment:
If the target field is given, I think it's fine to assume (from a documentation standpoint) that dedupe and preserve are set to true. I think it would be more clear to say:

For example, suppose the target field is set to 'is_target' and the following records are processed: